content/api/cli.md

```diff
@@ -1,8 +1,264 @@
+
+The `webpack` command is now available globally.
+
+### Common Usage
+
+``` sh
+webpack <entry> [<entry>] <output>
+```
+| Parameter | Configuration Mapping         | Explanation                                                                                                                                                             |
+|-----------|-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| entry     | entry                         | A filename or a set of named filenames which act as the entry point to build your project. If you pass a pair in the form of `=` you can create an additional entry point. |
+| output    | output.path + output.filename | A path and filename for the bundled file to be saved in.                                                                                                                |
+
+#### Examples
+
+If your project structure is as follows -
+```
+.
+├── dist
+├── index.html
+└── src
+    ├── index.js
+    ├── index2.js
+    └── others.js
+```
+
+`webpack src/index.js dist/bundle.js`
+	This will bundle your source code with entry as `index.js` and the output bundle file will have a path of `dist` and the filename will be `bundle.js`
+
+	| Asset     | Size    | Chunks      | Chunk Names |
+	|-----------|---------|-------------|-------------|
+	| bundle.js | 1.54 kB | 0 [emitted] | index       |
+		[0] ./src/index.js 51 bytes {0} [built]
+		[1] ./src/others.js 29 bytes {0} [built]
+
+`webpack index=./src/index.js entry2=./src/index2.js dist/bundle.js`
+	This will form the bundle with both the files as separate entry points.
+
+	| Asset     | Size    | Chunks        | Chunk Names   |
+	|-----------|---------|---------------|---------------|
+	| bundle.js | 1.55 kB | 0,1 [emitted] | index, entry2 |
+	[0] ./src/index.js 51 bytes {0} [built]
+	[0] ./src/index2.js 54 bytes {1} [built]
+	[1] ./src/others.js 29 bytes {0} {1} [built]
+
+### Common options
+
+<b>List all of the options available on the cli</b>
+``` sh
+webpack --help , webpack -h
+```
+
+<b>Build source using a config file</b>
+<p>Specifies a different configuration file to pick up. Use this if you want to specify something different than `webpack.config.js`, which is the default.</p>
+
+``` sh
+webpack --config example.config.js
+```
+
+<b>Send environment variable to be used in webpack config file</b>
+``` sh
+webpack --env=DEVELOPMENT
+```
+
+<b>Print result of webpack as a JSON</b>
+<p>In every other case, webpack prints out a set of stats showing bundle, chunk and timing details. Using this option the output can be a JSON object</p>
+<p>This response is accepted by webpack's [analyse tool](http://webpack.github.com/analyse). The analyse tool will take in the JSON and provide all the details of the build in graphical form</p>
+
+``` sh
+webpack --json , webpack -j, webpack -j > stats.json
```

Comment:
There's `--profile` flag that's related. This might be a good place to mention that. Or we could set up a "how to" for analyzing and link to there. That would work better as then we could link to other tools useful for analysis. I've done that [here](http://survivejs.com/webpack/building-with-webpack/analyzing-build-statistics/) if you want to see an example.